packages/custodyController/src/custody.ts

```diff
@@ -104,27 +104,26 @@ export class CustodyController {
     return custodyTypes;
   }
 
+  // TODO - Can probably be removed as that state is not being used anywhere
```

Comment:
Updated the method to use `envName`, but added TODO as this method can probably be removed.